CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.5-dev1
+
+* Added YoloX model for layout detection
```

Comment:
You can add this under the `0.2.4-dev1` heading. Given that we're adding a new model, I think we can justify a bump to `0.3.0`.